src/platforms/hosted/probe_info.h

```diff
@@ -48,6 +48,7 @@ typedef struct probe_info {
 	struct probe_info *next;
 } probe_info_s;
 
+char *extract_serial(const char *const device, const size_t length) ;
```

Comment:
clang-tidy will complain about the excess of const here (`const char *` and `size_t` only)

Reply:
Pushed update.